packages/dev/docs/pages/releases/2023-05-19.mdx

```diff
@@ -0,0 +1,81 @@
+{/* Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: In this release, we are moving TagGroup and Column Resizing to GA. We also have some fixes to Calendars and have been busy making many improvements to the new React Aria Components we announced last release.
+date: 2023-05-19
+---
+
+# May 19, 2023 Release
+
+In this release, we are moving TagGroup and Column Resizing to GA. We have also been making many improvements and bug fixes to the new [React Aria Components](../react-aria/react-aria-components.html) we announced last release. We have also added the much asked for Search to our docs. We hope this will aid in discoverability for our components and hooks.
+
+As always, thanks to all of our contributors for helping to make this release possible! Special shoutout to [@auskast](https://github.com/auskast) for an extensive PR to help us improve our React Aria Components.
+
+## New Components
+
+- [TagGroup](https://react-spectrum.adobe.com/react-spectrum/TagGroup.html)
```

Comment:
I know tableview isn't new and that it is mentioned in the description above, but do we wanna explicitly mention TableView column resizing GA in this section w/ a link to the section in the spectrum/aria docs as well?

Reply:
Yeah, i can do that